Gems/Maestro/Code/Source/Cinematics/CompoundSplineTrack.cpp

```diff
@@ -208,47 +210,44 @@ void CCompoundSplineTrack::SetValue(float time, const float& value, bool bDefaul
 }
 
 //////////////////////////////////////////////////////////////////////////
-void CCompoundSplineTrack::SetValue(float time, const Vec3& value, bool bDefault, bool applyMultiplier)
+void CCompoundSplineTrack::SetValue(float time, const AZ::Vector3& value, bool bDefault, bool applyMultiplier)
 {
     for (int i = 0; i < m_nDimensions; i++)
     {
-        m_subTracks[i]->SetValue(time, value[i], bDefault, applyMultiplier);
+        m_subTracks[i]->SetValue(time, value.GetElement(i), bDefault, applyMultiplier);
     }
 }
 
 //////////////////////////////////////////////////////////////////////////
-void CCompoundSplineTrack::SetValue(float time, const Vec4& value, bool bDefault, bool applyMultiplier)
```

Comment:
would need to test CCompoundSplineTrack and see if missing anything. quaternions between AZ and cry_quat are different. I forget how though?